src/main/java/firebase/sdk/dashboard/api/ReleaseResource.java

```diff
@@ -0,0 +1,175 @@
+package firebase.sdk.dashboard.api;
+
+import java.util.List;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.time.Instant;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import firebase.sdk.dashboard.data.SDK;
+import firebase.sdk.dashboard.data.SDKReleaseMetadata;
+import firebase.sdk.dashboard.data.Release;
+import firebase.sdk.dashboard.data.Platform;
+
+/**
+ * Root resource (exposed at "platforms/{platform}/releases" path)
+ */
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class ReleaseResource {
+
+  private String platform;
+  // TODO: Delete once actual implementation is done
+  private ArrayList<String> sdks = new ArrayList<>();
+
+  public ReleaseResource(String platform) {
+    this.platform = platform;
+    // Dummy Code for API
+    sdks.add("firebase-common");
+    sdks.add("firebase-common-ktx");
+    sdks.add("firebase-ml");
+    sdks.add("firebase-database");
+    sdks.add("firebase-auth");
+    sdks.add("firebase-components");
+  }
+
+  /**
+   * Method handling HTTP GET requests.
+   * Exposed at "v1/platforms/{platform}/releases", this endpoint returns a list
+   * of Release objects representing all the sdks enrolled in the given platform.
+   *
+   * @return Response object containing a list of Releease objects in JSON.
+   */
+  @GET
+  public Response getReleases() {
+    // TODO: implement this method
+    List<String> releaseManagers = Arrays.asList("ashwinraghav@", "rlazo@", "davidmotson@", "vkrtachko@", "vguthal@");
+    ArrayList<HashMap<String, Object>> releases = new ArrayList<HashMap<String, Object>>();
+    for (int i = 78; i > 0; i--) {
+      HashMap<String, Object> release = new HashMap<String, Object>();
+      release.put("releaseName", String.format("M%d", i));
+      release.put("platform", "Android");
+      release.put("releaseManager", releaseManagers.get(i % releaseManagers.size()));
+      release.put("launchCalDeadline", Instant.now());
+      release.put("codeFreezeTime", Instant.now());
+      release.put("launchDate", Instant.now());
+      release.put("buganizerHotlistLink", "b/12345678");
+      releases.add(release);
+    }
+
+    return ResponseHandler.createJsonResponse(Status.OK, releases);
+  }
+
+  /**
+   * Method handling HTTP POST requests.
+   * Exposed at "v1/platforms/{platform}/releases", this endpoint consumes a  
+   * Release object and adds it to the database. Only admins can do this.
+   *
+   * @return Response object containing a status code and a message to be 
```

Comment:
Do we need to send this message from server side? Looks like it can be derived from the status code.